### ISSUE TITLE
Olympus vsi files

### ISSUE THREAD

**Issue by Thecatinamustang** [BiomechDevOps/Histology-Image-Viewer-and-Converter#1]
Hey, 
I just stumbled upon your software and read it with high interest! 
However, i did not find any mentions on vsi files (from the olympus system), so i was wondering if these are covered, or if there is a way to get them implemented. I would be more than happy to help with providing images. ;)
At the moment i use Macros in imageJ with the OlympusViewer Plugin to convert my images in a userfrindly format to apply different techniqes to analyse my scan and to document my analyses. However, resolution is much lower, because the system crashes when i try to convert the highest resolution.

Long story short: Is there already a way to use your software on vsi files or will there be a way?

Best,

Katja Eildermann

**Comment by BiomechDevOps** [comment]
> Hey, I just stumbled upon your software and read it with high interest! However, i did not find any mentions on vsi files (from the olympus system), so i was wondering if these are covered, or if there is a way to get them implemented. I would be more than happy to help with providing images. ;) At the moment i use Macros in imageJ with the OlympusViewer Plugin to convert my images in a userfrindly format to apply different techniqes to analyse my scan and to document my analyses. However, resolution is much lower, because the system crashes when i try to convert the highest resolution.
> 
> Long story short: Is there already a way to use your software on vsi files or will there be a way?
> 
> Best,
> 
> Katja Eildermann

Hi Katja,

Thanks for your interest! Currently, our software doesn't support vsi files. We're open to exploring this, though. Could you send some vsi files to us for testing? Please email them to myself at srazian@unomaha.edu or Dr. Majid Jadidi at mjadidi@unomaha.edu. 

Thanks,
Sayed Ahmadreza Razian
